core/services/job/testdata/pretty.toml

```
@@ -0,0 +1,127 @@
+relay              = "evm"
+pluginType         = "median"
+contractID         = "foo"
+ocrKeyBundleID     = "bar"
+transmitterID      = "baz"
+contractConfigConfirmations = 1
+contractConfigTrackerPollInterval = "1s"
+
+[relayConfig]
+chainID = 1337
+fromBlock = 42
+
+[relayConfig.chainReader.chainContractReaders.median]
+contractABI = '''
+[
+  {
+    "anonymous": false,
+    "inputs": [
+      {
+        "indexed": true,
+        "internalType": "address",
+        "name": "requester",
+        "type": "address"
+      },
+      {
+        "indexed": false,
+        "internalType": "bytes32",
+        "name": "configDigest",
+        "type": "bytes32"
+      },
+      {
+        "indexed": false,
+        "internalType": "uint32",
+        "name": "epoch",
+        "type": "uint32"
+      },
+      {
+        "indexed": false,
+        "internalType": "uint8",
+        "name": "round",
+        "type": "uint8"
+      }
+    ],
+    "name": "RoundRequested",
+    "type": "event"
+  },
+  {
+    "inputs": [],
+    "name": "latestTransmissionDetails",
+    "outputs": [
+      {
+        "internalType": "bytes32",
+        "name": "configDigest",
+        "type": "bytes32"
+      },
+      {
+        "internalType": "uint32",
+        "name": "epoch",
+        "type": "uint32"
+      },
+      {
+        "internalType": "uint8",
+        "name": "round",
+        "type": "uint8"
+      },
+      {
+        "internalType": "int192",
+        "name": "latestAnswer_",
+        "type": "int192"
+      },
+      {
+        "internalType": "uint64",
+        "name": "latestTimestamp_",
+        "type": "uint64"
+      }
+    ],
+    "stateMutability": "view",
+    "type": "function"
+  }
+]
+'''
+
+[relayConfig.chainReader.chainContractReaders.median.chainReaderDefinitions]
```

Maybe something like this?
```suggestion
[relayConfig.chainReader.contracts.median.definitions]
```
```suggestion
[relayConfig.chainReader.contracts.median.configs]
```

---

Updated to `contracts` and `configs`